backend/routes/getters.js

```diff
@@ -0,0 +1,42 @@
+const { PrismaClient } = require("../generated/prisma");
+const prisma = new PrismaClient();
+
+const express = require("express");
+const app = express();
+app.use(express.json());
+const router = express.Router({ mergeParams: true });
+const yahooFinance = require("yahoo-finance2").default;
+
+router.get("/stats/:companyTick", async (req, res) => {
```

Comment:
should you check if the user is logged in in every end point here?
consider adding a middleware: https://stackoverflow.com/a/67618630